<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form2</class>
 <widget class="QWidget" name="Form2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Module music information</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../.designer/backup</normaloff>../../../../../.designer/backup</iconset>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="1" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>1</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>Brak odtwarzanej muzyki</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="modformatLabel">
        <property name="text">
         <string>Format MOD</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QFrame" name="okno"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>115</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>420</width>
       <height>115</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QPushButton" name="closeButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Zamknij</string>
      </property>
      <property name="shortcut">
       <string>Esc</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>30</y>
        <width>130</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Liczba sampli:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>130</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pattern:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>72</y>
        <width>130</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Liczba kanałów:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>93</y>
        <width>130</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Rows in pattern:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>51</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Liczba instrumentów:</string>
      </property>
     </widget>
     <widget class="QLabel" name="sampleLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>76</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="patternLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>76</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0 / 0</string>
      </property>
     </widget>
     <widget class="QLabel" name="rowLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>76</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0 / 0</string>
      </property>
     </widget>
     <widget class="QLabel" name="chanLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>76</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0 / 0</string>
      </property>
     </widget>
     <widget class="QLabel" name="instrLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>76</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Speed:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Tempo:</string>
      </property>
     </widget>
     <widget class="QLabel" name="spdLabel">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="bpmLabel">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>34</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
